### ISSUE TITLE
e2e and happo testing for errors

### ISSUE THREAD

**Issue by BCerki** [bcgov/cas-registration#1246]
###Description of the Tech Debt###

Some of the original e2e test writing cards didn't include checking validation errors. This card is about adding assertions and a happo shot for validation errors (both an empty submission and a submission that triggers incorrect formats). Note: if we do this in jest tests, consider whether it's also needed in e2e.

###Tech Debt Triage#
The purpose of our technical debt triage process is to analyze technical debt to determine risk level of the technical debt and the value in tackling that technical debt.

Risk Value Scoring:

| Level  | Value |
| ------ | ----- |
| High   | 3     |
| Medium | 2     |
| Low    | 1     |

| Technical Debt - Risk Types                                                                                                   | Level | Value |
| ----------------------------------------------------------------------------------------------------------------------------- | ----- | ----- |
| Business Area Risk - Risk of business area visibility / damage to user experience                                             |       |       |
| Developer Fault Risk - How likely will this tech debt cause a future error related to coding on top of it                     |       |       |
| System Fault Risk - Risk of system errors or application downtime                                                             |       |       |
| Time Scale Risk - Compound risk effect if left alone. How much more difficult to fix or dangerous will this become over time? |       |       |
| Time Sink Risk - How much will this tech debt slow the development process down                                               |       |       |
| TOTAL SCORE:                                                                                                                  |       |       |

**Comment by BCerki** [comment]
@patriciarussellCAS , I don't think we need this card anymore. We're sparingly testing with e2e, and we should already have validation errors covered by the integration tests

**Comment by patriciarussellCAS** [comment]
Thanks. I'll close it. If we need additional tickets I'd say, let's make new ones :)
